supplierProduct/tests.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from supplierProduct.models import supplierProduct, Product, Supplier
+
+supID0 = 123
+supID1 = 222
+supID2 = 127
+
+
+class TestSupProduct:
+
+    @pytest.fixture
+    def creat_product(self):
+        prod0 = Product(qr_code="AASSD22", product_name="APPLE", description="sweet!!")
+        prod1 = Product(qr_code="QS#$HHS", product_name="CARROT", description="yummy!!")
+        prod2 = Product(qr_code="AAsdgv66", product_name="BANANA", description="great!!")
+        prod0.save()
+        prod1.save()
+        prod2.save()
+        return [prod0, prod1, prod2]
+
+    @pytest.fixture
+    def creat_supplier(self):
+        sup0 = Supplier(
+            user_name="ali22",
+            first_name="ali",
+            last_name="ab",
+            password="2345678",
+            user_type="supplier")
+        sup1 = Supplier(
+            user_name="max00",
+            first_name="Max",
+            last_name="hani",
+            password="123456789",
+            user_type="supplier")
+        sup2 = Supplier(
+            user_name="rawanab",
+            first_name="rawan",
+            last_name="abu",
+            password="0000000",
+            user_type="supplier")
+        sup0.save()
+        sup1.save()
+        sup2.save()
+        return [sup0, sup1, sup2]
+
+    @pytest.fixture
+    def creat_supProd(self, creat_product, creat_supplier):
+        supprod0 = supplierProduct(
+            suppler_product_id=supID0,
+            user_name=creat_supplier[0],
+            qr_code=creat_product[0],
+            price=22, quantity=4)
```

Comment:
i think new supplier product that is created the quantity needs to be 0 in all

Reply:
why can't the supplier add a product with more then zero quantity  ?

Reply:
He can but in the test or by default when you are making a new supplier product the quantity automatically should be 0 unless the supplier increase or decrease

Reply:
I think it is OK to add a supplier_product with quantity>0.

Reply:
When creating quantity should be 1 and not 0,0 means delete the row because it is an extra row 

I dont see the logic when creating a supplier product the quantity can be anything really,supplier select product he start increasing,other wise there is no need for increase and decrease you can just use set method and thats it

Reply:
See my comment here https://github.com/redhat-beyond/BuyTogether/pull/38#discussion_r1058219817